<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mine Cop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="serverLabel">
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingButt">
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="serverButt">
          <property name="text">
           <string>Change Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logButt">
          <property name="text">
           <string>Start Logging</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="serverNameLabel">
        <property name="text">
         <string>servername</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>Log Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="fileList">
        <property name="maximumSize">
         <size>
          <width>120000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="playerLabel">
        <property name="text">
         <string>Players</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="playerList"/>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="locationLabel">
              <property name="text">
               <string>Location</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="locBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="saveLocationButt">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="locComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="filterLabel">
              <property name="text">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="filterBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
